### PR TITLE
FIX: Buttons in QR Scanner

### PR DESCRIPTION
**Pull request by marcosrdz**
Screenshot_1725210491

## Summary by CodeRabbit

- **New Features**
	- Introduced new visual indicators for flashlight status in the QR code scanning interface.
  
- **User Interface Enhancements**
	- Updated styles for camera control buttons to improve visibility and aesthetics.
	- Adjusted layout for better spacing and user experience in the camera controls.

### CODE REVIEW

File: scripts/react-native-camera-kit.patch [hunk 1]
```diff
@@ -9,3 +9,38 @@
                      if (zoomMode == "off") return true
                      val cameraControl = camera?.cameraControl ?: return true
                      val zoom = camera?.cameraInfo?.zoomState?.value?.zoomRatio ?: return true
+
+--- ../node_modules/react-native-camera-kit/dist/CameraScreen.js	2024-09-01 13:00:57
++++ ../node_modules/react-native-camera-kit/dist/CameraScreen.js	2024-09-01 13:00:46
```
Comment (Overtorment) [inline scripts/react-native-camera-kit.patch:14]: where is the patch applied? i just see a patch added, but dont  see a script that applies it

File: scripts/react-native-camera-kit.patch [hunk 2]
```diff
@@ -9,3 +9,38 @@
...
+         flexDirection: 'row',
+         justifyContent: 'space-between',
+-        paddingTop: 8,
+-        paddingBottom: 0,
++        paddingTop:44,
++        paddingHorizontal: 16,
+     },
+     cameraContainer: Object.assign({}, Platform.select({
+         android: {
+\ No newline at end of file
```
Comment (Overtorment) [inline scripts/react-native-camera-kit.patch:46]: what issue this patch solves?

can this patch be sent to upstream directly?

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: scripts/react-native-camera-kit.patch (hunk 1 above)
Comment (marcosrdz) [reply scripts/react-native-camera-kit.patch:14]: nom run patches

File: scripts/react-native-camera-kit.patch (hunk 2 above)
Comment (marcosrdz) [reply scripts/react-native-camera-kit.patch:46]: No. They are working on 14-beta. They no longer work on 13 releases

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**